index.ts

```diff
@@ -43,3 +46,25 @@ server.on('listening', (): void => {
     logger.info('http server listening on', conf.get('port') );
 });
 server.on('after', util.after);
+
+// Socket.IO
+if (conf.get('websocket.socket-io.enable')) {
+    var serverSio: any = restify.createServer(conf.get('serverOptions'));
```

Comment:
I need to expose one more property of the restify typing file in order to make it work. I will submit a PR for this change later.